test/types/lean.test.ts

```diff
@@ -75,6 +76,7 @@ async function gh11118(): Promise<void> {
   const UserModel = model<User>('User', schema);
 
   const docs = await UserModel.find().lean().exec();
+  if (!docs) return;
```

Comment:
This one is also unfortunately a red flag. `find()` can never return `null`, so something is incorrect here.